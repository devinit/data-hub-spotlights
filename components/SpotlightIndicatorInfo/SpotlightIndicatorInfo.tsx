import React, { FunctionComponent } from 'react';

interface SpotlightIndicatorInfoProps {
  heading?: string;
  description?: string;
}

const SpotlightIndicatorInfo: FunctionComponent<SpotlightIndicatorInfoProps> = props => {
  return (
    <>
      <div className="spotlight__subheading">{ props.heading }</div>
      <p className="spotlight__excerpt">{ props.description }</p>
    </>
  );
};

export { SpotlightIndicatorInfo };
