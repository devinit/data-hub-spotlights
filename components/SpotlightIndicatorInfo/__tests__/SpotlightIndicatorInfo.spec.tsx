/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { SpotlightIndicatorInfo } from '../SpotlightIndicatorInfo';

describe('SpotlightIndicatorInfo', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(
      <SpotlightIndicatorInfo heading="Heading" description="Description" />
    ).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
