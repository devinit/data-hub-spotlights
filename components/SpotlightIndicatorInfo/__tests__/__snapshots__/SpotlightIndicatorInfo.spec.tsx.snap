// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotlightIndicatorInfo renders correctly 1`] = `
Array [
  <div
    className="spotlight__subheading"
  >
    Heading
  </div>,
  <p
    className="spotlight__excerpt"
  >
    Description
  </p>,
]
`;
